corehq/apps/fixtures/fixturegenerators.py

```diff
@@ -61,11 +60,29 @@ def __call__(self, restore_state):
                 global_types[data_type._id] = data_type
             else:
                 user_types[data_type._id] = data_type
-        items = []
+        items = global_items = user_items = []
         if global_types:
-            items.extend(self.get_global_items(global_types, restore_state))
+            global_items = self.get_global_items(global_types, restore_state)
+            items.extend(global_items)
         if user_types:
-            items.extend(self.get_user_items(user_types, restore_user))
+            user_items = self.get_user_items(user_types, restore_user)
+            items.extend(user_items)
+
+        for metric_name, items_count in [
+            ('commcare.fixtures.item_lists.global', len(global_items)),
+            ('commcare.fixtures.item_lists.user', len(user_items)),
+            ('commcare.fixtures.item_lists.all', len(items)),
+        ]:
+            metrics_histogram(
+                metric_name,
+                items_count,
+                bucket_tag='items',
+                buckets=[1, 100, 1000, 10000, 30000, 100000, 300000, 1000000],
+                bucket_unit='',
+                tags={
+                    'domain': restore_user.domain
```

Comment:
are you sure you want to tag all domains?

Reply:
I think there's value in it? An alternative is to tag only on restores with more than 10,000 lookup table items, but then I might not be able to get a good impression of the full distribution for a given domain, just the distribution of their more lookup-table-intensive syncs. For example if we roll out a caching change, we won't be able to see the jump in a particular domain's number of restores with less than 1 (i.e. zero) lookup table items.

Any thoughts there? I suppose we could also remove half of the buckets and cut the metric count in half. Or require at least one sync with more than 30k lookup table items for a domain to be considered a "domain of interest" for this metric and then always report domains of interest, but that's getting a bit more complicated.